tests/Unit/Models/UserStoryTest.php

```
@@ -445,4 +445,77 @@ function (string $sql, array $params) use ($stmt, &$capturedSql, &$capturedParam
+
+    #[Test]
+    public function countUnresolvedAlignmentWarningsBatchReturnsMapByProjectId(): void
+    {
+        $capturedSql    = null;
+        $capturedParams = null;
+        $stmt = $this->createStub(\PDOStatement::class);
+        $stmt->method('fetchAll')->willReturn([
+            ['project_id' => 7, 'cnt' => 2],
+            ['project_id' => 8, 'cnt' => 0],
+        ]);
+        $db = $this->createStub(Database::class);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)